qa/suites/rbd/cache/workloads/qemu.yaml

```diff
@@ -0,0 +1,23 @@
+tasks:
+- install:
+    extra_packages: ['ceph-immutable-object-cache']
+- ceph:
+    conf:
+      osd:
+        osd shutdown pgref assert: true
+      client:
+        rbd_parent_cache_enabled: true
+        #debug immutable_obj_cache: 20/20
+        immutable object cache path: /tmp/ceph-immutable-object-cache
+        immutable object cache max size: 50G
+      global:
+        auth cluster required: none
+        auth service required: none
+        auth client required : none
+- qemu:
+    client.0:
+      clone: true
+      test: qa/workunits/true.sh
```

Comment:
I think it would be great if you had a workload that actually needed the "golden" image to ensure it works.